packages/client/tests/functional/issues/18970-invalid-date/tests.ts

```diff
@@ -0,0 +1,71 @@
+import { getQueryEngineProtocol } from '@prisma/internals'
+
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+testMatrix.setupTestSuite(
+  () => {
+    testIf(getQueryEngineProtocol() === 'graphql')('throws on invalid date (graphql)', async () => {
+      await expect(
+        prisma.user.findMany({
+          where: {
+            date: new Date('I am not a date'),
+          },
+        }),
+      ).rejects.toMatchPrismaErrorInlineSnapshot(`
+
+        Invalid \`prisma.user.findMany()\` invocation in
+        /client/tests/functional/issues/18970-invalid-date/tests.ts:0:0
+
+          XX () => {
+          XX   testIf(getQueryEngineProtocol() === 'graphql')('throws on invalid date (graphql)', async () => {
+          XX     await expect(
+        → XX       prisma.user.findMany({
+                     where: {
+                       date: new Date('Invalid Date')
+                       ~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Is it expected that the squiggly line doesn't cover the whole line or is shifted from the value towards the key?